src/kontrol/hevm.py

```diff
@@ -0,0 +1,34 @@
+from __future__ import annotations
+
+from typing import TYPE_CHECKING
+
+from pyk.kast.inner import KApply
+
+if TYPE_CHECKING:
+    from pyk.kast.inner import KInner
+
+
+class Hevm:
+    @staticmethod
+    def help_info() -> list[str]:
+        res_lines: list[str] = []
+        print_foundry_success_info = any('hevm_success' in line for line in res_lines)
+        if print_foundry_success_info:
+            res_lines.append('')
+            res_lines.append('See `hevm_success` predicate for more information:')
+            res_lines.append(
+                'https://github.com/runtimeverification/kontrol/blob/master/src/kontrol/kdist/hevm.md#hevm-success-predicate'
+            )
+        res_lines.append('')
+        res_lines.append(
+            'Access documentation for KEVM foundry integration at https://docs.runtimeverification.com/kontrol'
```

Comment:
Sorry, another nitpick: I guess, it's not really Foundry integration if we're not using Foundry's success predicate? Maybe we can say "documentation for KEVM and hevm integration"? or just "documentation for Kontrol"? I think we can probably just switch the _"Access documentation for Kontrol at ..."_ for both `foundry` and `hevm` modes :) 
wdyt?